Version.props

```diff
@@ -1,14 +1,18 @@
 <Project ToolsVersion="15.0">
-  <PropertyGroup Condition=" '$(PackageVersion)' == '' ">
-    <PackageReleaseVersion>1.0.1</PackageReleaseVersion>
-    <PackageBuildQuality>$(BUILD_QUALITY)</PackageBuildQuality>
-    <PackageBuildQuality Condition=" '$(PackageBuildQuality)' == '' ">beta1</PackageBuildQuality>
-    <PackageVersion>$(PackageReleaseVersion)-$(PackageBuildQuality)</PackageVersion>
+  <PropertyGroup Condition=" '$(PackageVersion)' == '' ">	
+    <!-- If $(PB_VersionStamp) is empty and $(PB_IsStable) is not true, then a default PackageBuildQuality will be added. -->
+    <PackageReleaseVersion Condition="'$(PackageReleaseVersion)' == '' ">1.0.1</PackageReleaseVersion>
+    <PackageBuildQuality>$(PB_VersionStamp)</PackageBuildQuality>
+    <PackageBuildQuality Condition=" '$(PackageBuildQuality)' == '' And '$(PB_IsStable)' != 'true' ">beta1</PackageBuildQuality>
```

Comment:
do we need to update the default value to preview2?

Reply:
This has been beta1 pretty much forever, we can update it, but I don't think it's strictly required

Reply:
FYI this means that the build quality will be 'preview2' for 2.1 preview2 rather than 'beta2'.  Sound good? Otheriwse you can handle as you wish.